lib/elixir_analyzer/exercise_test/assert_call/compiler.ex

```diff
@@ -195,4 +244,35 @@ defmodule ElixirAnalyzer.ExerciseTest.AssertCall.Compiler do
   """
   def in_function?(name, {_module_path, name}), do: true
   def in_function?(_, _), do: false
+
+  @doc """
+  Extract paths from AST path, considering branching from calls like `import A.B.{C, D.E}`
+  and return a list of pairs of alias and full path
+  """
+  def get_paths({:__aliases__, _, path}, :import),
+    do: [{path, path}]
+
+  def get_paths({{:., _, [root, :{}]}, _, branches}, :import) do
+    [{root_path, _}] = get_paths(root, :import)
+
+    for branch <- branches,
+        {path, _} <- get_paths(branch, :import) do
+      {root_path ++ path, root_path ++ path}
+    end
+  end
+
+  def get_paths({:__aliases__, _, path}, :alias),
+    do: [{[List.last(path)], path}]
+
+  def get_paths({{:., _, [root, :{}]}, _, branches}, :alias) do
+    [{_, root_path}] = get_paths(root, :alias)
+
+    for branch <- branches,
+        {last, full_path} <- get_paths(branch, :alias) do
+      {last, root_path ++ full_path}
+    end
+  end
+
+  def get_paths(path, _type) when is_atom(path),
+    do: [{[path], [path]}]
```

Comment:
This, like above, seems to service two use cases and I think they are being merged together because they are similar, but they are a little bit different and that makes them harder to understand since it is hard to know if they can jump between the two (I suspect they can't but it appears that way on inspection).  Suggestion: separate them into two function `get_import_paths` and `get_alias_paths`